src/command/KeyBindingManager.js

```diff
@@ -410,6 +411,13 @@ define(function (require, exports, module) {
             });
         }
         
+        // Also make sure that the second word of PageUp/PageDown has the first letter in upper case.
+        if (/^Page/.test(key)) {
+            key = key.replace(/(up|down)$/, function (match, p1) {
+                return p1.charAt(0).toUpperCase() + p1.substr(1);
+            });
+        }
```

Comment:
It looks like this changes `Pageup` to be `PageUp` and `Pagedown` to be `PageDown`, but `pageup` or `pagedown` are not changed. Is that correct?